src/main/ctl/git.ts

```diff
@@ -96,7 +111,7 @@ class Git {
     // Prompt window to input username && password, and use https protocol
     const username = await host.showInputBox({
       placeHolder: "Username",
-      prompt: "Please input your username of GitHub",
+      prompt: `Please input your username of ${this.showContent}`,
```

Comment:
If you use `gitProvider`, just replace `showContent` to `capitalize(gitProvider)`.